kentik_api_library/kentik_api/public/errors.py

```diff
@@ -0,0 +1,29 @@
+# This is temporary solution until comprehensive error handling is in place
+
+
+class DataFormatError(Exception):
+    """ When KentikAPI HTTP response JSON has field of invalid type """
+
+
+class DeserializationError(Exception):
```

Comment:
Isn't DeserializationError and DataFormatError too specific? I think it would be better to merge them into more general `ResponseDataError`. Difference between them is blurry for me, because deserialization errors might be caused by invalid data format and corrupted JSON could be considered a data format error.

Reply:
We can use ResponseDataError. Please provide such exception type in exception handling delivery.